### PR TITLE
Fix Domain Screen Minor Issues

### PR DESCRIPTION
**Pull request by ItzNotABug**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

**Comment by ItzNotABug** [comment]
@TorstenDittmann should we also move to using `https` protocol for the domains? [Currently its `http`](https://github.com/appwrite/console/blob/16ee13934d1ae175a8eecadb6c890aaf00dd79bd/src/lib/pages/domains/index.svelte#L73).

**Review by TorstenDittmann — APPROVED** [review]

**Comment by TorstenDittmann** [comment]
> @TorstenDittmann should we also move to using `https` protocol for the domains? [Currently its `http`](https://github.com/appwrite/console/blob/16ee13934d1ae175a8eecadb6c890aaf00dd79bd/src/lib/pages/domains/index.svelte#L73).

yes, lets do that 👍🏻

**Comment by ItzNotABug** [comment]
> > @TorstenDittmann should we also move to using `https` protocol for the domains? [Currently its `http`](https://github.com/appwrite/console/blob/16ee13934d1ae175a8eecadb6c890aaf00dd79bd/src/lib/pages/domains/index.svelte#L73).
> 
> yes, lets do that 👍🏻

Done ✅

File: src/lib/helpers/load.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export enum View {
 export function getView(url: URL, route: Page['route'], fallback: View): View {
     return (url.searchParams.get('view') ?? preferences.get(route).view) === View.Grid
         ? View.Grid
-        : View.Table ?? fallback;
+        : (View.Table ?? fallback);
```
Comment (TorstenDittmann) [inline src/lib/helpers/load.ts:24]: I think this changes logic. Fallback will never be reached.

Comment (ItzNotABug) [reply src/lib/helpers/load.ts:24]: Leftover from `pnpm format`. Merging main should be fine.

**Pull request merged**